src/phases/attempt-capture-phase.ts

```diff
@@ -137,7 +137,7 @@ export class AttemptCapturePhase extends PokemonPhase {
                 repeatDelay: 500,
                 onUpdate: (t) => {
                   if (shakeCount && shakeCount < (isCritical ? 2 : 4)) {
-                    const value = t.getValue();
+                    const value = t.getValue() ?? 1;
```

Comment:
```suggestion
                    const value = t.getValue() ?? 0;
```
Same here (is this just recycled code?)

Reply:
Yeah MEs copy/pasted the capture code.